pkg/utils/managedresources/registry.go

```diff
@@ -23,18 +24,21 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
-	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	apimachineryjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
```

Comment:
```suggestion
	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
```

See https://github.com/gardener/gardener/blob/c3a26ebaf23e18fe61a7dd822f76f708b1505c09/pkg/admissioncontroller/webhook/admission/auditpolicy/handler_test.go#L32